cmd/release/cmd/update.go

```diff
@@ -42,8 +45,101 @@ var updateK3sReferencesCmd = &cobra.Command{
 	},
 }
 
+var updateChartsCmd = &cobra.Command{
+	Use:     "charts [branch] [chart] [version]",
+	Short:   "Update charts files locally, stage and commit the changes.",
+	Example: "release update charts 2.9 rancher-istio 104.0.0+up1.21.1",
+	Args: func(cmd *cobra.Command, args []string) error {
+		if err := validateChartConfig(); err != nil {
+			log.Fatal(err)
+		}
+
+		if len(args) != 3 {
+			return errors.New("expected 3 arguments: [branch] [chart] [version]")
+		}
+
+		var branch, chart, version string
+		var found bool
+		var err error
```

Comment:
This can be a problem when an error is checked and the action is not to return which leaves this value potentially untouched on a subsequent check causing issues. Let's not define a catch-all error value here.

Reply:
Implemented